client/app/bundles/course/survey/pages/ResponseShow/__test__/index.test.jsx

```diff
@@ -18,10 +13,11 @@ describe('<ResponseShow />', () => {
     const spyFetch = jest.spyOn(CourseAPI.survey.responses, 'fetch');
 
     // Mount response show page and wait for data to load
-    Object.defineProperty(window.location, 'pathname', { value: responseUrl });
     mount(
-      <ResponseShow params={{ courseId, surveyId, responseId }} />,
-      generateContextOptions(storeCreator({}))
+      <MemoryRouter initialEntries={[responseUrl]}>
+        <ResponseShow {...{ survey: {}, courseId, surveyId, match: { params: { responseId } } }} />
+      </MemoryRouter>,
+      buildContextOptions(storeCreator({}))
```

Comment:
After nesting the pages in `SurveyLayout`, only two of the tests use `MemoryRouter`. Just removed the other occurrences.